events/views.py

```diff
@@ -219,3 +225,22 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id, slug):
+    """
+        Filter evets for category
+    """
+    try:
+        events = CategoryEvent.objects.filter(category_id=id)
+    except CategoryEvent.DoesNotExist:
+        raise Http404("Selected category nonexistent")
```

Comment:
Using the localization string "_".

``raise Http404(_("Selected category nonexistent"))``